core/src/test/scala/scala/pickling/registry/SwitchRuntimeRegistryInit.scala

```diff
@@ -0,0 +1,46 @@
+package scala.pickling.registry
+
+import org.scalatest.FunSuite
+import scala.pickling._
+import scala.pickling.internal.HybridRuntime
+import scala.pickling.internal.DefaultRuntime
+import scala.pickling.json.JsonFormats
+import scala.pickling.pickler.AllPicklers
+
+object Protocol extends {
+  val oldRuntime = internal.currentRuntime
+  val currentRuntime = new HybridRuntime
+  val onlyLookup = internal.replaceRuntime(currentRuntime)
+} with Ops with AllPicklers with JsonFormats
+
+class SwitchRuntimeRegistryInit extends FunSuite {
```

Comment:
Sure, I'll do that. Also, I'll try to figure out how to test something that it's not in the same project automatically when `test` is run.